mesheryctl/internal/cli/root/system/channel.go

```diff
@@ -164,7 +172,15 @@ var switchCmd = &cobra.Command{
 // Switch between release channels
 mesheryctl system channel switch [stable|stable-version|edge|edge-version]
 	`,
-	Args: cobra.ExactArgs(1),
+	Args: func(_ *cobra.Command, args []string) error {
+		const errMsg = `Usage: mesheryctl system channel switch [stable|stable-version|edge|edge-version]
+Example: mesheryctl system channel switch stable`
+
+		if len(args) == 0 {
+			return fmt.Errorf("accepts single argument, received %d\n\n%v", len(args), errMsg)
```

Comment:
cmd.help()? 

Also, Should we handle the case where there will be more than 1 args given to the function?